### DISCUSSION TITLE
Mach3 GrblHal conversion - Spindle & more

### DISCUSSION THREAD

**Discussion by Switch-Witch** [grblHAL/core#557] [General]
I want to replace the mach3 Bob of my cnc with the T41U5XBB and there are some questions left I can`t answer myself.
Is there anybody who could be so kind to give me some hints?

To connect the spindle to the mach3 board I used "analogue ground, 0-10V analogue, 10V inverted output, com and X1 (forward).
0-10V and ground seems to be clear but where to connect the 10V inverted, the com and the X1 (forward)?

many thx in advance for any help

![Bildschirmfoto 2024-07-14 um 18 17 20](https://github.com/user-attachments/assets/eec0fe8d-7f56-45a3-8a68-b9708d734364)

<img width="346" alt="Bildschirmfoto 2024-07-15 um 10 47 37" src="https://github.com/user-attachments/assets/97e2b0a1-c5d5-4349-adf6-5c7457faf8a0">

**Comment by terjeio** [comment]
Com is G (Gnd), X1 is likely Dir.
10V from the inverter is not needed, or it could be connected to the 12V input if you do not have a separate supply for that.

**Reply by Switch-Witch** [reply]
Thx so much!! 😘
You saved my life 😄
